linux/deploy_vm/set_new_vm_user_account.yml

```diff
@@ -23,3 +23,4 @@
 - name: "Get VM user password hash"
   ansible.builtin.set_fact:
     vm_password_hash: "{{ vm_password | password_hash('sha512') }}"
+    vm_password_md5: "{{ vm_password | password_hash('md5') }}"
```

Comment:
ubuntu supports sha512 check sum, why not use the sha512 password hash?

Reply:
At firstly I use  {{ vm_password_hash }} and I can't login to the OS. Thanks.